config/ember-try.js

```diff
@@ -48,8 +48,8 @@ module.exports = function () {
           name: 'ember-lts',
           npm: {
             devDependencies: {
-              'ember-source': '~3.16.0',
-              'ember-data': '~3.16.0',
+              'ember-source': '~3.24.0',
+              'ember-data': '~3.24.0',
```

Comment:
Let's do these changes in a dedicated PR, so we can have a nice entry in the changelog.

Reply:
done